CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# CHANGELOG
+
+## 0.5.0
```

Comment:
```suggestion
# 0.5.0
```
I recommend we change this to match the bottlerocket-os changelog.  No need to call out that it's a changelog again.

(Would also have to remove a "#" from 0.4.0 below.)